src/plugins/locale/index.ts

```diff
@@ -144,8 +123,7 @@ export const localePlugin: EsDayPlugin<{}> = (_, dayClass, dayFactory) => {
   const oldEndOf = dayClass.prototype['endOf']
   const fixDiff = (inst: EsDay, origin: EsDay, unit: UnitType, reverse = false) => {
     if (prettyUnit(unit) === C.WEEK) {
-      // default start of week is Monday
-      const defaultStartOfWeek = C.INDEX_MONDAY
```

Comment:
IMHO '4' is not thursday, but the 4th of january - the first week of the year is the week that contains the 4th of january.
See [moment docs](https://momentjs.com/docs/#/customization/dow-doy/) or [wikipedia section 'First week'](https://en.wikipedia.org/wiki/ISO_week_date)